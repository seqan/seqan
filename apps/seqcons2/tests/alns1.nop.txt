>consensus_0
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 GGTGTGTCGGCCCTATCCCTTGCGCTTACAGACTATATTTCTTCGTGAGGGAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCG  GTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGC
          GCCCTATCCCTTGCGCTTACAGACTTTATTTCTTCGTGAGGGAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAG                ATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTATGATCCAGTTGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGA
                    TTGCGCTTACAGACTTTATTTCTTCGTGAGGGAGTTGACCCATGCAGAACCAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCACCAG          CGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTCTGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTC
                                                   GAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAA  TACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCAGGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCA
                                                     GTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACC         CTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGG
                                                           CCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCA    TAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGC
                                                                            TTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTAT  TCCAGTAGGTCCACGGAATCAAGGCCGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
                                                                                       TGGGAAAATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTATGATCCAGTAGG      GAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGCAGCTTGGGGGGTAGCCGGCAACGGCCCGGCTAACAGC
                                                                                                                                                                      GGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAA
                                                                                                                                                                          CGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
                                                                                                                                                                             TATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATTCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
                                                                                                                                                                                           TCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
