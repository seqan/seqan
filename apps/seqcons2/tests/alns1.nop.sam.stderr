SeqCons
=======

__OPTIONS____________________________________________________________________

VERBOSITY              	NORMAL
OPERATION              	convert only

INPUT FILE             	alns1.sam
OUTPUT READS           	alns1.nop.sam
OUTPUT CONSENSUS       	alns1.nop.fa

OVERLAP MIN LENGTH     	20
OVERLAP MAX ERR-RATE   	5
OVERLAP MIN COUNT      	3
OVERLAP WINDOW SIZE    	20

K-MER SIZE             	20
K-MER MAX OCCURRENCES  	200

REALIGNMENT BANDWIDTH  	10
REALIGNMENT ENVIRONMENT	20

__LOADING DATA_______________________________________________________________

  Loading alignments from alns1.sam...OK

__COMPUTATION________________________________________________________________


__WRITING RESULT_____________________________________________________________

Writing consensus to alns1.nop.fa ... OK
Writing alignments to alns1.nop.sam ... OK

Overall time: 0.00347531 s

Done.  Have a nice day.
